lib/rules/lint-require-iframe-title.js

```diff
@@ -28,12 +37,23 @@ module.exports = class IframeHasTitle extends Rule {
           } else if (!title.value) {
             return;
           } else if (title.value.type === 'TextNode') {
-            if (title.value.chars.trim().length === 0) {
+            let value = title.value.chars.trim();
+            if (value.length === 0) {
               this.logNode({
                 message: errorMessage,
                 node,
               });
               return;
+            } else if (this.knownTitles.includes(value)) {
+              this.logNode({
+                message:
+                  errorMessage +
+                  ' ' +
+                  `Value title="${title.value.chars}" already used for different iframe.`,
```

Comment:
Do you think we could keep track of both the preexisting titles **and** their locations? I think it might be nice to include the location that we found the alternate title in this error message. What do you think?

Reply:
yeah, it make sense, will do it

Reply:
done